product-service-description/compliance-as-a-service/features/TSE-Epson-Interoperabilität.md

```diff
@@ -25,6 +27,10 @@ Das _fiskaltrust.Middleware.SCU.DE.Epson_-Package ermöglicht die Verbindung der
 - **Port:** Port des Epson Servers, Printers oder lokjalen Webservices, üblicherweise 8009
 - **DeviceId:** Diese Einstellung ist nur für den Epson Server relevant und dient der Identifikation des Servers. Die DeviceId kann im Epson Server Dashboard gefunden werden, welches über die Host IP Adresse aufgerufen werden kann.
 
+## Troubleshooting:
+
+Wenn die Developer- TSE einen Selbsttest verlangt obwohl sie aus- und eingesteckt wurde, führen Sie bitte einen Factory-Reset gemäß den Hersteller-Angaben durch.
```

Comment:
We should add that this is only possible for dev-tses

Reply:
improved communication